READMEs/parameters.md

```diff
@@ -7,58 +7,45 @@ SPDX-License-Identifier: Apache-2.0
 
 We can set any config parameter (a) via an envvar, or (b) via a config file. Envvar values override config file values.
 
-An `Ocean` instance will hold a `Config` instance that holds various config parameters. These parameters need to get set. This is set based on what's input to `Ocean` constructor:
+An `Ocean` instance will hold a `Config` instance that holds various config parameters. These parameters can be set in a variety of ways:
 
-1.  dict input: `Ocean({'network':..})`
-2.  Config object input: `Ocean(Config('config.ini'))`
-3.  no input, so it uses OCEAN_CONFIG_FILE envvar
+1. Config object input
+    * Filled from ExampleConfig: `Ocean(ExampleConfig.get_config())` (recommended)
+    * Filled from config file: `Ocean(Config('config.ini'))`
+    * Filled from config file specified by `OCEAN_CONFIG_FILE` envvar: `Ocean(Config())`
+    * Filled from config dict: `Ocean(Config('network':..}))`
+2.  dict input: `Ocean({'network':..})` (deprecated because it doesn't support all config options)
 
 Here are examples.
 
-## 1. dict input, filled from envvars
+## 1a. Config object input, filled from ExampleConfig (recommended)
 
 First, in console:
 
 ```console
 export OCEAN_NETWORK_URL=https://rinkeby.infura.io/v3/<your Infura project id>
-export METADATA_CACHE_URI=https://aquarius.rinkeby.oceanprotocol.com
```

Comment:
Here you deleted the exports of `METADATA_CACHE_URI` and `PROVIDER_URL` and for `Unsetting envvars` section you basically unset something that it's not mentioned at the beginning, maybe we can keep those methods as we had before and just add another method for config creation such as ExampleConfig to include all of the aspects.